test/game_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'simplecov'
+SimpleCov.start
+
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/game"
+require "pry"
+
+class GameTest < Minitest::Test
+
+  def setup
+    @base_game = Game.new({:game_id => 123,
```

Comment:
I made up non-sensical values but they are the same data-type as they should be. Let me know if you think this could create an issue.